README.md

```diff
@@ -119,15 +119,19 @@ python -m deeppavlov <mode> <path_to_config> [-d]
+* Add section to [*utils/configs/model_info.json*](utils/configs/model_info.json) with your custom Telegram messages
+* In model config file specify `metadata.labels.telegram_utils` parameter with name which refers to the added section of [*utils/configs/model_info.json*](utils/configs/model_info.json)
 
-For the `interactmsbot` mode you should specify **Microsoft app id** in `-i` and **Microsoft app secret** in `-s`. Also before launch you should specify api deployment settings (host, port) in [*utils/server_config.json*](utils/server_utils/server_config.json) configuration file. Note, that Microsoft Bot Framework requires `https` endpoint with valid certificate from CA.
-Here is [detailed info on the Microsoft Bot Framework integration](http://docs.deeppavlov.ai/en/latest/devguides/ms_bot_integration.html) 
+For the `interactmsbot` mode you should specify **Microsoft app id** in `-i` and **Microsoft app secret** in `-s`. Also before launch you should specify api deployment settings (host, port) in [*utils/configs/server_config.json*](utils/configs/server_config.json) configuration file. Note, that Microsoft Bot Framework requires `https` endpoint with valid certificate from CA.
+Here is [detailed info on the Microsoft Bot Framework integration](http://docs.deeppavlov.ai/en/latest/devguides/ms_bot_integration.html)
 
-For `riseapi` mode you should specify api settings (host, port, etc.) in [*utils/server_config.json*](utils/server_utils/server_config.json) configuration file. If provided, values from *model_defaults* section override values for the same parameters from *common_defaults* section. Model names in *model_defaults* section should be similar to the class names of the models main component.
+You can also store your tokens, app ids, secrets in appropriate sections of [*utils/configs/server_config.json*](utils/configs/server_config.json). Please note, that all command line parameters override corresponding config ones.
+
+For `riseapi` mode you should specify api settings (host, port, etc.) in [*utils/configs/server_config.json*](utils/configs/server_config.json) configuration file. If provided, values from *model_defaults* section override values for the same parameters from *common_defaults* section. Model names in *model_defaults* section should be similar to the class names of the models main component.
 Here is [detailed info on the DeepPavlov REST API](http://docs.deeppavlov.ai/en/latest/devguides/rest_api.html)
 
+All DeepPavlov configuration files are stored in `utils/configs` by default. You can get full path to it with `python -m deeppavlov.config configs`. Also you can move it with with `python -m deeppavlov.config configs -p <new/configs/dir/path>` (all your configuration settings will be preserved) or move it to default location with `python -m deeppavlov.config configs -d` (all your configuration settings will be RESET to default ones).
```

Comment:
Naming is really close to pipelines configs, it's confusing. How do we distinguish them?

Reply:
Fixed